brainstormforce/astra
# AST-1413 - Footer alignment option for dark-mode not working in preview

### Description
- Alignment option preview support added for footer dark mode element
- Fixed alignment issue with frontend > footer > dark mode

### Screenshots
- https://share.bsf.io/NQu7AdKD

### Types of changes
- Bug fix 

### How has this been tested?
- Check footer dark mode > alignment option
- Check with free theme
- Chekc with pro advanced options

### Checklist:
